app/components/RadioPlayer.vue

```diff
@@ -414,15 +426,15 @@ export default {
       const ogTitle = document.querySelector("meta[property='og:title']")
       document.title = ogTitle ? ogTitle.getAttribute('content') : 'Lahmacun radio'
 
-      /* Google tags
-            if (this.show_check) {
-                gtag('event', 'Radio play', {
-                    'event_category': this.show_title,
-                    'event_label': 'Play state',
-                    'value': 0,
-                });
-            }
-            */
+      // Google Analytics 3 stop
+      if (this.show_check) {
+        // eslint-disable-next-line no-undef
+        gtag('event', 'Radio play', {
+          event_category: this.show_title,
+          event_label: this.show_subtitle,
+          value: 0
+        })
+      }
```

Comment:
@baxgas for some time ago we decided to get rid of stop events because they confuse the stats. Can you delete it here too?